README.rst

```diff
@@ -8,30 +8,37 @@ Example use
 
 .. code-block:: python
 
-   from zpl import Label
-
-   l = Label(65,60)
-   height = 0
-   l.origin(0,0)
-   l.write_text("Problem?", char_height=10, char_width=8, line_width=60, 
-   justification='C')
-   l.endorigin()
-
-   height += 13
-   image_width = 55
-   l.origin((l.width-image_width)/2, height)
-   image_height = l.write_graphic(Image.open('trollface-large.png'), 
-   image_width)
-   l.endorigin()
-
-   l.origin(0, height+image_height)
-   l.write_text('Happy Troloween!', char_height=5, char_width=4, 
-   line_width=60,
-                justification='C')
-   l.endorigin()
-
-   print(l.dumpZPL())
-   l.preview()
+    from zpl import label 
+
+    l = Label(100,60)
+    height = 0
+    l.origin(0,0)
+    l.write_text("Problem?", char_height=10, char_width=8, line_width=60, justification='C')
+    l.endorigin()
+
+    height += 13
+    image_width = 5
+    l.origin((l.width-image_width)/2, height)
+    image_height = l.write_graphic(
+        Image.open(os.path.join(os.path.dirname(__file__), 'trollface-large.png')),
+        image_width)
+    l.endorigin()
+
+    height += 13
+    l.origin(0, height)
+    l.write_bar_code(height=150, barcode_type='2')
+    l.write_text('123456', char_height=5, char_width=4, line_width=60,
+                 justification='C')
+    l.endorigin()
+
+    height += image_height
+    l.origin(0, height)
+    l.write_text('Happy Troloween!', char_height=5, char_width=4, line_width=60,
+                 justification='C')
+    l.endorigin()
+
+    print(l.dumpZPL())
+    l.preview()
 
 This will display the following preview image, generated using the `Labelary API <http://labelary.com/>`_:
```

Comment:
this line will need to be updated to also contain the newly added barcode.

Reply:
Check latest commits, changed the raw ZPL and labelary call